generators/chipyard/src/main/scala/Clocks.scala

```diff
@@ -58,18 +24,12 @@ object GenerateReset {
   def apply(chiptop: ChipTop, clock: Clock): Reset = {
     implicit val p = chiptop.p
     // this needs directionality so generateIOFromSignal works
-    val reset_wire = Wire(Input(Reset()))
-    val (reset_io, resetIOCell) = p(GlobalResetSchemeKey) match {
-      case GlobalResetSynchronous =>
-        IOCell.generateIOFromSignal(reset_wire, "reset")
-      case GlobalResetAsynchronousFull =>
-        IOCell.generateIOFromSignal(reset_wire, "reset", abstractResetAsAsync = true)
-      case GlobalResetAsynchronous => {
-        val async_reset_wire = Wire(Input(AsyncReset()))
-        reset_wire := ResetCatchAndSync(clock, async_reset_wire.asBool())
-        IOCell.generateIOFromSignal(async_reset_wire, "reset", abstractResetAsAsync = true)
-      }
-    }
+    val async_reset_wire = Wire(Input(AsyncReset()))
+    val (reset_io, resetIOCell) = IOCell.generateIOFromSignal(async_reset_wire, "reset",
+      abstractResetAsAsync = true)
+
+    val reset_wire = ResetCatchAndSync(clock, async_reset_wire.asBool())
```

Comment:
@davidbiancolin We don't need this reset synchronizer anymore now that we have the diplomatic synchronizer, right?

Reply:
Can you just double check in the generated verilog that the only fanout of this async reset is to the catch and sync?

Reply:
Yeah. The async reset coming out of the IOCell drives only the the catchandsyncs in the ResetSynchronizer node.